resources/fetchers/ecr_fetcher.go

```diff
@@ -34,8 +35,11 @@ import (
 )
 
 const (
-	PrivateRepoRegexTemplate = "^%s\\.dkr\\.ecr\\.%s\\.amazonaws\\.com\\/([-\\w\\.\\/]+)[:,@]?"
+	PrivateRepoRegexTemplate = "^%s\\.dkr\\.ecr\\.([-\\w]+)\\.amazonaws\\.com\\/([-\\w\\.\\/]+)[:,@]?"
 	PublicRepoRegex          = "public\\.ecr\\.aws\\/\\w+\\/([-\\w\\.\\/]+)\\:?"
```

Comment:
can you please add some documentation about this regex, what are we expecting it to parse, and an example string or something?